client/components/mma/cancel/CancellationJourneyFunnel.tsx

```diff
@@ -36,9 +33,20 @@ export const CancellationJourneyFunnel = () => {
 		return <Navigate to="/" />;
 	}
 
+	const productType = getSpecificProductTypeFromTier(productDetail.tier);
+	const productTypeKey = productType.productType;
+
+	const possiblePaidPlan = productDetail.subscription.currentPlans[0];
+	const qmEventString = `cancellation start | ${productType.friendlyName}${
+		isPaidSubscriptionPlan(possiblePaidPlan)
+			? ` | billing period: ${possiblePaidPlan.billingPeriod}`
+			: ''
+	}`;
+	window.QuantumMetricAPI?.sendEvent(184, 0, qmEventString);
```

Comment:
what is 184 and 0? (and should they be named consts?)

Reply:
yeah, I've renamed them based on the interface you found 👍